admin/securedrop_admin/__init__.py

```diff
@@ -642,7 +642,7 @@ def check_for_updates(args):
 
 
 def get_release_key_from_keyserver(args, keyserver=None, timeout=45):
-    gpg_recv = ['timeout', str(timeout), 'gpg', '--recv-key']
+    gpg_recv = ['timeout', str(timeout), 'gpg', '--batch', '--no-tty', '--recv-key']
```

Comment:
it seems like the `-x` option was not used when cherry-picking this commit?

Reply:
This wasn't a cherry-pick. I had to add this to get the lint to pass.